### PR TITLE
[DEV-7357] Add clear selection

### PR DESCRIPTION
**Pull request by aspirisen**
DEV-7357

### CODE REVIEW

File: packages/slate-commons/src/commands/resetSelection.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import type { Editor } from 'slate';
+import { Transforms } from 'slate';
+
+import { findLeafLocation } from './findLeafLocation';
+import { isValidLocation } from './isValidLocation';
+
+export function resetSelection(editor: Editor) {
+    const locationLeaf = location && findLeafLocation(editor, [0]);
```
[the remark this answers is not in the document]
Comment (aspirisen) [reply packages/slate-commons/src/commands/resetSelection.ts:8]: From here https://github.com/prezly/slate/blob/main/packages/slate-commons/src/commands/saveSelection.ts#L22

[the remark this answers is not in the document]
Comment (e1himself) [reply packages/slate-commons/src/commands/resetSelection.ts:8]: But there `location` is an argument of the function. Here, it refers to the `window.location` global.

[the remark this answers is not in the document]
Comment (aspirisen) [reply packages/slate-commons/src/commands/resetSelection.ts]: Oops, thanks)

**Review by e1himself — APPROVED** [review]

File: packages/slate-commons/src/commands/resetSelection.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import { findLeafLocation } from './findLeafLocation';
+import { isValidLocation } from './isValidLocation';
+
+export function resetSelection(editor: Editor) {
+    const locationLeaf = findLeafLocation(editor, [0]);
+
+    if (locationLeaf && isValidLocation(editor, locationLeaf)) {
+        Transforms.select(editor, locationLeaf);
+        Transforms.collapse(editor, { edge: 'start' });
```
Comment (e1himself) [inline packages/slate-commons/src/commands/resetSelection.ts:12]: Can we combine these into one operation?

**Pull request merged**